src/main/java/com/commercetools/sync/commons/helpers/BaseSyncStatistics.java

```diff
@@ -81,6 +81,10 @@ public void incrementCreated() {
     created.incrementAndGet();
   }
 
+  /** decrements the total number of resources that failed to sync. */
+  public void decrementFailed() {
```

Comment:
I agree here. This is because you incrementFailed when some category is waitingToBeResolved... is this really the case?

Reply:
yes   , the problem is that I have to increment failed, when a parent is not resolvable, because I cannot check if the parent will be imported in another batch. And when then the parent is incremented in a future batch, I have to "reset" the failed statistic.